launch/tier4_simulator_launch/package.xml

```diff
@@ -17,6 +17,7 @@
   <exec_depend>autoware_dummy_perception_publisher</exec_depend>
   <exec_depend>autoware_elevation_map_loader</exec_depend>
   <exec_depend>autoware_fault_injection</exec_depend>
+  <exec_depend>autoware_launch</exec_depend>
```

Comment:
@isamu-takagi and maintainers
It feels a bit strange that a package in autoware_universe depends on autoware_launch.
But is this something that can't be helped?

Reply:
It seems that the config file in autoware_launch has an original config file in a package in autoware_universe.
For example:
```
$(find-pkg-share autoware_launch)/config/perception/traffic_light_recognition/traffic_light_fine_detector/traffic_light_fine_detector.param.yaml
$(find-pkg-share autoware_traffic_light_fine_detector)/config/traffic_light_fine_detector.param.yaml
```
So I think this can be resolved by specifying all the configuration files by arguments and referencing the universe package by default.

Reply:
I agree, we should remove the dependency on autoware_launch from tier4_simulator_launch, and the config files should be passed by arguments.

Reply:
@YamatoAndo @mitsudome-r 
OK, I'll create a separate PR to remove the dependency.

Reply:
https://github.com/autowarefoundation/autoware_universe/pull/10404/commits/bad11b5dbe2906ee61a4870de152878d6fdcaf90